tools/accuracy_checker/accuracy_checker/metrics/speech_recognition.py

```diff
@@ -0,0 +1,70 @@
+"""
+Copyright (c) 2019 Intel Corporation
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import numpy as np
+
+from ..config import BoolField, NumberField, ConfigError
+from ..representation import (
+    CharacterRecognitionAnnotation,
+    CharacterRecognitionPrediction,
+    )
+from .metric import PerImageEvaluationMetric
+from ..utils import finalize_metric_result
+
+
+class SpeechRecognitionWER(PerImageEvaluationMetric):
+    __provider__ = 'wer'
+    annotation_types = (CharacterRecognitionAnnotation,)
+    prediction_types = (CharacterRecognitionPrediction,)
+
+    overall_metric = []
```

Comment:
no need to specify here if you have it in configure (this method call inside __init__())

Reply:
fixed